packages/icon-builder/package.json

```diff
@@ -16,27 +21,30 @@
     "React",
     "react-component"
   ],
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/callemall/material-ui-icons.git"
-  },
-  "author": "Hai Nguyen",
-  "license": "MIT",
-  "dependencies": {
-    "yargs": "^4.3.2"
+  "scripts": {
+    "prebuild": "rimraf build",
+    "build": "npm run build:icons && npm run build:copy-files",
+    "build:icons": "node ./build.js --output-dir ./build --svg-dir ./node_modules/material-design-icons --glob '/**/production/*_24px.svg' --renameFilter ./filters/rename/material-design-icons.js",
+    "build:copy-files": "babel-node ./copy-files.js",
+    "test": "grunt"
   },
-  "peerDependencies": {},
   "devDependencies": {
-    "babel-cli": "^6.6.5",
-    "grunt": "^0.4.5",
+    "babel-cli": "^6.0.0",
```

Comment:
We used [yarn](https://github.com/yarnpkg/yarn) over npm to install our dependencies on cicleci. I can take care of that.